tests/globalconfig.pm

```diff
@@ -34,108 +34,110 @@ BEGIN {
     use base qw(Exporter);
 
     our @EXPORT = qw(
-        $anyway
-        $automakestyle
-        $CURL
-        $CURLVERSION
-        $CURLVERNUM
-        $DATE
-        $has_shared
-        $LIBDIR
-        $UNITDIR
-        $TUNITDIR
-        $SRVDIR
-        $listonly
-        $LOCKDIR
-        $LOGDIR
-        $memanalyze
-        $MEMDUMP
-        $perlcmd
-        $perl
-        $PIDDIR
-        $proxy_address
-        $PROXYIN
-        $pwd
-        $randseed
-        $run_duphandle
-        $run_event_based
-        $SERVERCMD
-        $SERVERIN
-        $srcdir
-        $TESTDIR
-        $torture
-        $valgrind
-        $VCURL
-        $verbose
-        %feature
-        %keywords
-        @protocols
-        $bundle
-        $dev_null
+      $anyway
+      $automakestyle
+      $CURL
+      $CURLVERSION
+      $CURLVERNUM
+      $DATE
+      $has_shared
+      $LIBDIR
+      $UNITDIR
+      $TUNITDIR
+      $SRVDIR
+      $listonly
+      $LOCKDIR
+      $LOGDIR
+      $memanalyze
+      $MEMDUMP
+      $perlcmd
+      $perl
+      $PIDDIR
+      $proxy_address
+      $PROXYIN
+      $pwd
+      $randseed
+      $run_duphandle
+      $run_event_based
+      $SERVERCMD
+      $SERVERIN
+      $srcdir
+      $TESTDIR
+      $torture
+      $valgrind
+      $VCURL
+      $verbose
+      %feature
+      %keywords
+      @protocols
+      $bundle
+      $dev_null
     );
 }
 use pathhelp qw(
-    exe_ext
-    dirsepadd
+  exe_ext
+  dirsepadd
 );
 use Cwd qw(getcwd);
 use testutil qw(
-    shell_quote
+  shell_quote
 );
 use File::Spec;
 
-
 #######################################################################
 # global configuration variables
 #
 
 # config variables overridden by command-line options
-our $verbose;         # 1 to show verbose test output
-our $torture;         # 1 to enable torture testing
-our $proxy_address;   # external HTTP proxy address
-our $listonly;        # only list the tests
-our $run_duphandle;   # run curl with --test-duphandle to verify handle duplication
-our $run_event_based; # run curl with --test-event to test the event API
-our $automakestyle;   # use automake-like test status output format
-our $anyway;          # continue anyway, even if a test fail
-our $CURLVERSION="";  # curl's reported version number
-our $CURLVERNUM="";   # curl's reported version number (without -DEV)
+our $verbose;    # 1 to show verbose test output
+our $torture;    # 1 to enable torture testing
+our $proxy_address;    # external HTTP proxy address
+our $listonly;    # only list the tests
+our
+  $run_duphandle; # run curl with --test-duphandle to verify handle duplication
+our $run_event_based;    # run curl with --test-event to test the event API
+our $automakestyle;    # use automake-like test status output format
+our $anyway;    # continue anyway, even if a test fail
+our $CURLVERSION = "";    # curl's reported version number
+our $CURLVERNUM = "";    # curl's reported version number (without -DEV)
```

Comment:
I'd be nicer/readable IMO to keep the original vertical aligment.
Perhaps there is an option to leave all comments as-is?